static/app/utils/replays/hooks/useReplayData.spec.tsx

```diff
@@ -32,10 +32,14 @@ jest.mocked(useProjects).mockReturnValue({
   placeholders: [],
 });
 
+const mockInvalidateQueries = jest.fn();
```

Comment:
Can we reset this before or after each test? This way if another test gets added the state from a previous test won't affect the new one.

Reply:
Ah yes good point, added in the existing `beforeEach`.